### PR TITLE
Support Travis CI

### PR DESCRIPTION
**Pull request by pine**
I want to test this library in Travis CI.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -1,4 +1,3 @@
 #!/bin/sh
-xbuild
-nunit-console GeoHexTest/bin/Debug/GeoHexTest.dll
-
+xbuild /p:Configuration=Debug GeoHex.sln
+mono ./packages/NUnit.ConsoleRunner.3.2.1/tools/nunit3-console.exe GeoHexTest/bin/Debug/GeoHexTest.dll
```
Comment (pine) [inline test.sh:3]: Should be used nunit-console v3.x

File: GeoHexTest/GeoHexTest.csproj [hunk 2]
```diff
@@ -42,7 +42,7 @@
       <HintPath>..\packages\NUnit.3.2.1\lib\net45\nunit.framework.dll</HintPath>
     </Reference>
     <Reference Include="GeoHex, Version=0.0.1, Culture=neutral, PublicKeyToken=2638cd05610744eb">
-      <HintPath>..\GeoHex\bin\Debug\GeoHex.dll</HintPath>
+      <HintPath>..\GeoHex\bin\$(Configuration)\GeoHex.dll</HintPath>
```
Comment (pine) [inline GeoHexTest/GeoHexTest.csproj:45]: `$(Configuration)` is Debug or Release.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+language: csharp
+solution: GeoHex.sln
+mono:
+  - 4.2.3
```
Comment (pine) [inline .travis.yml:4]: Fix version by the current stable version.

File: GeoHexTest/GeoHexTest.cs [hunk 4]
```diff
@@ -222,6 +222,11 @@ private List<string[]> ParseCsv(string path)
             return wordsList;
         }
 
+        private string GetTestFilePath(string path)
+        {
+            return System.IO.Path.Combine(TestContext.CurrentContext.TestDirectory, "../..", path);
```
Comment (mattak) [inline GeoHexTest/GeoHexTest.cs:227]: Thanks! It looks nice.

File: GeoHexTest/GeoHexTest.csproj (hunk 2 above)
Comment (mattak) [inline GeoHexTest/GeoHexTest.csproj:45]: 💯

**Comment by mattak** [comment]
Thanks you for awesome patch. :-)
But I cannot test in my mac environment... 
Do you build this branch with windows?

**Comment by mattak** [comment]
Sorry. My nuget packages path was located on `~/.nuget/packages`.
So the build was failed...
When I change location setting, it works well.

So I'll merge the pull request.

**Comment by pine** [comment]
🎉 🎉 🎉 🎉 🎉 🎉

**Pull request merged**